### PR TITLE
updates to closed PRs

### PR DESCRIPTION
**Pull request by kognate**
This adds the world clock,  news, and weather widgets.

fixes some typos in other text.

### CODE REVIEW

File: distance_display/README.md [hunk 1]
```diff
@@ -13,9 +13,18 @@ this repository.
 
 ![Widget In Action](distance_widget.png)
 
-There are three user-adjustable parameters in the widget.  The first, `how_many_to_show` sets the number of countries shown in the widget.  The second, `output_format` defines the distance output format.  This can be in miles,  feet, meters, kilometers, or rods.  The last parameter, `near_or_far` tells the widget to sort by nearest or most distant.
-
-The code uses the haversine distance equation to calculate distances.  There is also a javascript implmentation in the file `haversine.js`
+There are three user-adjustable parameters in the widget.  The first,
+`how_many_to_show` sets the number of countries shown in the widget.
+The second, `output_format` defines the distance output format.  This
```
Comment (jeanlange) [inline distance_display/README.md]: `output_format` => `output_format`,

File: distance_display/README.md [hunk 2]
```diff
@@ -13,9 +13,18 @@ this repository.
...
+can be in miles, feet, meters, kilometers, or rods.  The last
+parameter, `near_or_far` tells the widget to sort by nearest or most
```
Comment (jeanlange) [inline distance_display/README.md]: `near_or_far` => `near_or_far`,

File: distance_display/README.md [hunk 3]
```diff
@@ -13,9 +13,18 @@ this repository.
 
 ![Widget In Action](distance_widget.png)
 
-There are three user-adjustable parameters in the widget.  The first, `how_many_to_show` sets the number of countries shown in the widget.  The second, `output_format` defines the distance output format.  This can be in miles,  feet, meters, kilometers, or rods.  The last parameter, `near_or_far` tells the widget to sort by nearest or most distant.
-
-The code uses the haversine distance equation to calculate distances.  There is also a javascript implmentation in the file `haversine.js`
```
Comment (jeanlange) [inline distance_display/README.md]: ooh, I am pleased by your changes to the second paragraph. Links & spelling correction FTW!

File: distance_display/README.md (hunk 1 above)
Comment (kognate) [inline distance_display/README.md]: I don't understand what you mean here.

File: distance_display/README.md (hunk 2 above)
Comment (kognate) [inline distance_display/README.md]: I don't understand what you mean here.

**Pull request merged**